config/src/test/kotlin/org/springframework/security/config/web/server/ServerOneTimeTokenLoginDslTests.kt

```diff
@@ -146,6 +153,48 @@ class ServerOneTimeTokenLoginDslTests {
         // @formatter:on
     }
 
+    @Test
+    fun `oneTimeToken when custom token expiration time set then authenticate`() {
+        spring.register(OneTimeTokenConfigWithCustomTokenExpirationTime::class.java).autowire()
+
+        // @formatter:off
+        client.mutateWith(SecurityMockServerConfigurers.csrf())
+                .post()
+                .uri{ uriBuilder: UriBuilder -> uriBuilder
+                        .path("/ott/generate")
+                        .build()
+                }
+                .contentType(MediaType.APPLICATION_FORM_URLENCODED)
+                .body(BodyInserters.fromFormData("username", "user"))
+                .exchange()
+                .expectStatus()
+                .is3xxRedirection()
+                .expectHeader().valueEquals("Location", "/login/ott")
+
+        client.mutateWith(SecurityMockServerConfigurers.csrf())
+                .post()
+                .uri{ uriBuilder:UriBuilder -> uriBuilder
+                        .path("/ott/generate")
+                        .build()
+                }
+                .contentType(MediaType.APPLICATION_FORM_URLENCODED)
+                .body(BodyInserters.fromFormData("username", "user"))
+                .exchange()
+                .expectStatus()
+                .is3xxRedirection()
+                .expectHeader().valueEquals("Location", "/login/ott")
+
+        val token = TestServerOneTimeTokenGenerationSuccessHandler.lastToken
```

Comment:
Will you please make [the same repair](https://github.com/spring-projects/spring-security/commit/751b5580a129f9b3a1da02c7f49bffd6ed35ef32) here in the same preceding commit?